.github/workflows/nightly-builds.yml

```diff
@@ -65,6 +65,7 @@ jobs:
     runs-on: ubuntu-20.04
     if: github.repository == 'akka/akka'
     strategy:
+      fail-fast: false
```

Comment:
add that in binary-compatibility-checks.yml also

Reply:
done